client/my-sites/comments/comment-navigation/index.jsx

```diff
@@ -230,7 +267,11 @@ const mapStateToProps = ( state, { commentsPage, siteId } ) => {
 			};
 		}
 	} );
-	return { visibleComments };
+
+	return {
+		visibleComments,
+		isCommentsTreeSupported: ! isJetpackSite( state, siteId ) || isJetpackMinimumVersion( state, siteId, '5.3' ),
```

Comment:
It's fine to leave this here, but just a note that I don't believe we've updated the client to use comments-tree for Jetpack sites yet.